debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -19,10 +23,20 @@
  */
 public class CommonConnectorConfig {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(CommonConnectorConfig.class);
+
     public static final int DEFAULT_MAX_QUEUE_SIZE = 8192;
     public static final int DEFAULT_MAX_BATCH_SIZE = 2048;
     public static final long DEFAULT_POLL_INTERVAL_MILLIS = 500;
 
+    public static final Field TASKS_MAX = Field.create("tasks.max")
+            .withDisplayName("Maximum tasks per connector")
+            .withType(Type.INT)
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.LOW)
+            .withDescription("Maximum tasks that can be started by connector, only '0' or '1' is allowd")
```

Comment:
s/allowd/allowed/

Does 0 even make sense? Seems fixing it to 1 would be more useful

Reply:
@gunnarmorling You are right, I've checked the Connect code and 1 is the minimum